### DISCUSSION TITLE
Crop image base on bounding boxes

### DISCUSSION THREAD

**Discussion by AnchorAnalytics** [albumentations-team/autoalbument#10] [Q&A]
My goal is to crop an imaged based on its bounding boxes coordinates.  Is there a crop function in albumentations that vertically or horizontally crops an image based on x,y coordinates.  See image for example?

Thanks! 

![cat_LI](https://user-images.githubusercontent.com/68970725/107117942-be54da80-684b-11eb-86a1-857b64601759.jpg)

**Comment by Dipet** [comment]
The same issue here https://github.com/albumentations-team/albumentations/issues/838#issuecomment-774474735
